exporter/sumologicexporter/config.go

```diff
@@ -64,6 +64,10 @@ type Config struct {
 	SourceHost string `mapstructure:"source_host"`
 	// Name of the client
 	Client string `mapstructure:"client"`
+
+	// StickySessionEnabled defines if sticky session support is enable.
+	// By default this is false.
+	StickySessionEnabled bool `mapstructure:"sticky_session_enabled"`
```

Comment:
If you potentially have further settings related to sticky session I would make this into a section

Reply:
It would be breaking change for us for now as we use it for some time already, so we would like to merge it as it is for now and eventually migrate it if there will be a need

Reply:
Ah, didn't get that this was already part of the Sumologic distro. Thanks for clarifying!